src/main/java/ru/sbtqa/tag/pagefactory/PageFactory.java

```diff
@@ -172,4 +183,46 @@ public static Environment getEnvironment() {
                 }
         }
     }
+
+    private static void initializeFieldsMap() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 21 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS3776)